fa25-team-a/eda/README.md

```
@@ -0,0 +1,50 @@
+* Pipeline:
+   1. Load Excel file
+   2. Explore sheets and pick the most completed and updated sheet as the dataset to be cleaned
+   3. Explore dataset: data types, null values, all columns
+        * Note all findings in a Markdown table
+   4. Outline next steps to manipulate the dataset: questions to ask clients, cleaning methods, and creating new columns
+        * Will perform more EDA using clarification from the Oct 10 client meeting
+        * perform **geocoding** and **race–ethnicity realignment** before advanced modeling or equity analysis.
+* Notebooks:
+   * `EDA_Chapter-40B_applicant_2023-25.ipynb`
+      * `zip` null values can be extracted from `matched_address`
+      * further geocode to fill the rest of `matched_address `
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 4; Actual: 6

(MD007, ul-indent)